### ISSUE TITLE
Travis possibly running tests in parallel

### ISSUE THREAD

**Issue by jsymons** [rmotr-group-projects/pyp-w4-gw-twitter-timeline#25]
We need to see if Travis is running the different versions of Python in parallel. Had issues where students' tests were failing randomly on Travis while passing on tox. When manually restarting the individual failing build jobs they passed just fine. Only thing I could come up with was that the different versions were testing in parallel. Since the tests each drop all the database values as part of their setup if two tests are running at the same time they can cause erroneous results.

**Comment by santiagobasulto** [comment]
It probably is running things in parallel. I'll investigate.

**Comment by santiagobasulto** [comment]
Sorry, didn't mean to close the Issue. I took a look and found the way to set the number of concurrent jobs in Travis. Already updated it as shown in the screenshot below. Is there anyway to test this @jsymons? Any of the students' PRs that you know it's working but travis is failing?

![image](https://cloud.githubusercontent.com/assets/872296/25225091/b0b316ea-2596-11e7-8a89-56c7bf788533.png)

https://docs.travis-ci.com/user/customizing-the-build#Limiting-Concurrent-Builds

**Comment by jsymons** [comment]
I'll put in a pr later today to test with. I don't want to restart any of the currently passing ones as there's another issue that may cause them to fail that I can't fix without access to their files. (Dependency issue introduced by new version of ipython being released yesterday)

**Comment by jsymons** [comment]
Ok ran a PR through and it's no longer running in parallel. This issue shouldn't pop up again.
